packages/components/inputs/selectable-search-input/src/selectable-search-input.tsx

```diff
@@ -212,11 +212,13 @@ const defaultProps: Pick<
   | 'isClearable'
   | 'menuHorizontalConstraint'
   | 'showSubmitButton'
+  | 'menuPortalZIndex'
```

Comment:
I think we could also generate new README for this package, as we're missing this prop description there

Reply:
thanks, updated [5ab8bda](https://github.com/commercetools/ui-kit/pull/2452/commits/5ab8bda266e4797cc86382c852f9a4ad0925026f)